examples/distgen/README.rst

```diff
@@ -0,0 +1,261 @@
+
```

Comment:
Should this file be part of the current PR? I think it only belongs in #583

Reply:
No, agreed.